### PR TITLE
Mad 8836 - Build Portal API Documentation: CloudServer Module

### PR DESCRIPTION
**Pull request by ppf**

### CODE REVIEW

File: cloud-server/resize.md [hunk 1]
```diff
@@ -1,32 +1,27 @@
...
 
-**Parameters**:
+#### Input
 - int `package_id` (required): the id of the service package to resize to. @see package:list
 
-**Request**:
+#### Request:
 ```
-curl -i -X POST "$PORTAL_API_URL/v1/cloud-server/$CLOUDSERVER_ID/resize" \
+$ curl -i -X POST "$PORTAL_API_URL/v1/cloud-server/{id}/resize" \
```
Comment (adrian-enspired) [inline cloud-server/resize.md:16]: use `$VARIABLES`

File: cloud-server/show.md [hunk 2]
```diff
@@ -1,26 +1,22 @@
...
-
-**Access**: service view permission
-
-**Parameters**: none
-
-**Request**:
+#### Request:
 ```
-curl -i "$PORTAL_API_URL/v1/cloud-server/$CLOUDSERVER_ID" \
+$ curl -i "$PORTAL_API_URL/v1/cloud-server/{id}" \
```
Comment (adrian-enspired) [inline cloud-server/show.md:14]: use `$VARIABLES`

File: cloud-server/start.md [hunk 3]
```diff
@@ -1,29 +1,25 @@
...
 
-**Parameters**: none
+#### Input
+none
 
-**Request**:
+#### Request:
 ```
-curl -i -X POST "$PORTAL_API_URL/v1/cloud-server/$CLOUDSERVER_ID/start" \
+$ curl -i -X POST "$PORTAL_API_URL/v1/cloud-server/{id}/start" \
```
Comment (adrian-enspired) [inline cloud-server/start.md:16]: use `$VARIABLES`

File: cloud-server/stop.md [hunk 4]
```diff
@@ -1,29 +1,25 @@
...
 
-**Parameters**: none
+#### Input
+none
 
-**Request**:
+#### Request:
 ```
-curl -i -X POST "$PORTAL_API_URL/v1/cloud-server/$CLOUDSERVER_ID/stop" \
+$ curl -i -X POST "$PORTAL_API_URL/v1/cloud-server/{id}/stop" \
```
Comment (adrian-enspired) [inline cloud-server/stop.md:16]: use `$VARIABLES`

**Review by adrian-enspired — CHANGES_REQUESTED** [review]

File: cloud-server/add.md [hunk 5]
```diff
@@ -23,20 +17,19 @@ This task is queued, meaning it will be completed out-of-band from the current r
 - integer[] `ssh_key_ids` (required if `use_password`=false): id of the ssh key(s) to install on the new cloud server. @see ssh-key:list
 - integer[] `addon_ids` (optional): ids of service add-on packages to order
 - string `discount_code` (optional): discount code to apply to the order
-
-**Additional Parameters**:
 - string `designated_ip` (optional): public IP address to assign to the new cloudserver. The IP must be available to OpenStack and not already be in use. NOTE, this functionality is exposed only via the admin `launch:as-client` endpoint.
```
[the remark this answers is not in the document]
Comment (ppf) [reply cloud-server/add.md:20]: ok, removing

**Review by adrian-enspired — APPROVED** [review]

**Pull request merged**